library-backend/library-app/src/main/java/org/internship/library/app/controller/BookController.java

```diff
@@ -0,0 +1,59 @@
+package org.internship.library.app.controller;
+
+import org.internship.library.api.Book;
+import org.internship.library.api.BookService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(path = "book")
+public class BookController {
+
+    Logger logger = LoggerFactory.getLogger(BookController.class);
+
+    @Autowired
+    BookService bookService;
+
+    /**
+     * Receive id of a book
```

Comment:
The javadoc should say something like below.

The first line should always say what will that method do.
Then, each @param or @return tag should describe its role.
In our case it's pretty simple and redundant but nevertheless it's a good exercise and something get used to. Please update for all of the methods in `BookService`, `BookServiceImpl` and `BookController`.

```java
/**
 * Retrieves the book specified by the id
 * 
 * @param id the book id
 * @return the book
 */
```